sklearn/naive_bayes.py

```diff
@@ -212,6 +212,8 @@ class labels known to the classifier.
     [1]
     """
 
+    _parameter_constraints = {"priors": ["array-like", None], "var_smoothing": [Real]}
```

Comment:
Can you update the param description in the docstring, adding that the default is None: 
`priorsarray-like of shape (n_classes,), default=None` ?

Reply:
Updated the docstring for the `priors` parameter for the Gaussian NB class